src/src/Pokemon.java

```diff
@@ -0,0 +1,66 @@
+public abstract class Pokemon {
+    String name;
```

Comment:
probeer je properties private te maken.
Anders kan je in je Main.java waar je nu je pokemon aanmaakt ook dit doen:

charmender.name = "Pickachu";

terwijl je eigenlijk er voor wilt zorgen dat je properties alleen te zetten zijn via je constructor of via je setters.

Nu lijkt dit voor een kleine applicatie niet zo heel belangrijk omdat je hier alleen aan werkt, maar als je in een groot team werkt of een API bouwt voor een externe partij is het heel belangrijk dat je 100% ownership hebt over wat er kan en mag binnen je class. Daarom zou ik altijd je properties private maken zodat je zelf je business rules kan bedenken binnen je setter functies. (bijv. dat de lengte van een naam min 4 characters mag hebben)